packages/server/src/migration/migs/index.ts

```diff
@@ -0,0 +1,3 @@
+import { UserTokenMigration } from './0.2.0-user-token'
+
+export default [UserTokenMigration]
```

Comment:
This is where we put our migrations. There is no folder listing etc. It could be done any other way but this works and is pretty simple.